model/movies.py

```diff
@@ -1,13 +1,16 @@
+from blueprints.movies import init_cache
+
 class Movies():
     def __init__(self):
+        init_cache()
```

Comment:
Itt a cinitet be kellene állítani, különben újra inicializálni fognak a fenti függvényeid.